src/java/org/apache/cassandra/db/ReadCommand.java

```diff
@@ -124,7 +125,8 @@ public abstract class ReadCommand extends AbstractReadQuery
     private final boolean isDigestQuery;
     private final boolean acceptsTransient;
     private final Epoch serializedAtEpoch;
-    private boolean allowsOutOfRangeReads;
+    private final boolean allowsPotentialTxnConflicts;
```

Comment:
not a blocking comment, would be good to make this an enum rather than `boolean`

Reply:
I converted it to an enum